docs/dev/ci.md

```diff
@@ -103,6 +103,23 @@ To skip tasks on TaskCluster add the following to the title of your pull request
 
 This should only be used for metadata files, such as those in `.github`, `LICENSE` or `CODE_OF_CONDUCT.md`.
 
+### Release-like Android builds
+
+Release builds are done on [TaskCluster].
+As it builds for multiple platforms this task is quite long and does not run on pull requests by default.
+
+It is possible trigger the task on pull requests.
+Add the following literal string to the pull request **title**:
```

Comment:
Interesting, I always thought these flags should be added to the commit message or body. Or is that the case for `[doc only]` but not the case for `[ci full]`?

Reply:
It's true for all the others (because CircleCI looks at commit messages).
But TaskCluster is (currently) built to look at the PR.

We _could_ unify that at some point, but for now we copied this behavior from others and didn't adjust it.